src/lib/bump-stats/index.ts

```diff
@@ -2,8 +2,7 @@ import * as Sentry from '@sentry/electron/main';
 import { isSameDay, isSameMonth, isSameWeek } from 'date-fns';
 import fetch from 'node-fetch';
 import { loadUserData, saveUserData } from 'src/storage/user-data';
-
-export type AggregateInterval = 'daily' | 'weekly' | 'monthly';
+import { AggregateInterval, StatsGroup, StatsMetric } from './types';
```

Comment:
Should we use an absolute path?
```suggestion
import { AggregateInterval, StatsGroup, StatsMetric } from 'src/lib/bump-stats/types';
```